src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -273,7 +277,7 @@ int64_t ClusterResourceScheduler::GetBestSchedulableNode(const TaskRequest &task
     // of soft constraint violations.
```

Comment:
Github won't let me add this comment on line 264, but can you add a TODO before the placement hint case that says it isn't sampled at uniform random?

Reply:
Done.